framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -179,6 +179,9 @@ public void generateReports(List<DrillTest> tests, int iteration) {
       if(drillReportsDir == null){
         drillReportsDir = CWD;
       }
+      if(drillReportsDFSDir == null){
```

Comment:
Why check here, not at the place NPE is thrown? You just need to have one check right?